packages/main/src/types/MessageStripDesign.ts

```diff
@@ -26,6 +26,18 @@ enum MessageStripDesign {
 	 * @public
 	 */
 	Warning = "Warning",
+
+	/**
+	* Message is custom
```

Comment:
Maybe change it to - Message uses custom Color Set 1, or something which differentiates between the two enum values